webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleAdvancedFilterDrawer.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+  <exo-drawer
+      ref="peopleAdvancedFilterDrawer"
+      id="peopleAdvancedFilterDrawer"
+      right
+      @closed="close">
+    <template slot="title">
+      <span class="popupTitle">{{ $t('pepole.advanced.filter.title') }}</span>
+    </template>
+    <template slot="content">
+      <v-form class="pa-2 ms-2 mt-4">
+      <div class="d-flex flex-column flex-grow-1">
+      <div class="d-flex flex-row" v-for="item in settings" :key="item">
+        <input
+            v-model="item.valueToSearch"
+            :placeholder="getResolvedName(item)"
+            type="text"
+            class="input-block-level ignore-vuetify-classes my-3">
+      </div>
+      </div>
+      </v-form>
+    </template>
+    <template slot="footer">
+      <div class="VuetifyApp flex d-flex">
+        <v-btn
+            class="peopleAdvancedFilterResetButton"
+            @click="reset">
+          <v-icon x-small class="pr-1">fas fa-redo</v-icon>
+          <template>
+            {{ $t('pepole.advanced.filter.button.reset') }}
+          </template>
```

Comment:
Useless `template` element
```suggestion
          {{ $t('pepole.advanced.filter.button.reset') }}
```

Reply:
fixed